Build-and-Deploy-Your-First-Dashboard.mdx

```diff
@@ -0,0 +1,420 @@
+To view this connection, please go to the **Connect** tab as seen below:
+
+
+<Frame>
+<img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/data-connection-hero.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/data-connection-hero-dark.png"
+  alt="Hero Dark"
+/>
+</Frame>
+
+If you would like to connect to a different data source, please have a look at how to [set up your connection](data-connection/data-connection).
+
+
+## Build your first dashboard
+
+In order to build your first dashboard you should begin by creating your first chart before adding to the dashboard. 
+
+
+### Build your first chart
+
+It requires five steps to build your first chart:
+
+<AccordionGroup>
+  <Accordion title="1. Select raw table(s)">
+
+    Search and select the **aggregations** table. 
+
+    <Frame>
+<img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/select-raw-tables.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/select-raw-tables-dark.png"
+  alt="Hero Dark"
+/>
+</Frame>
+    <Tip>
+
+    The list of available raw table(s) is generated after the [data connection](/data-connection/data-connection) is set up.
+
+    </Tip>
+
+  </Accordion>
+
+{" "}
+<Accordion title="2. Prepare data">
+Click on the **+ aggregations** , this will create ```SQL SELECT * FROM "aggregations" ```   
+in the SQL editor.
+
+Click on **Preview data** to run the SQL and **Next** once you see the preview data.
+     <Frame>
+
+  <img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/query-builder.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/query-builder-dark.png"
+  alt="Hero Dark"
+/>
+    </Frame>
+
+
+ <Tip>
+    If you are more advanced with SQL, you can add more tables and customise your SQL.
+    </Tip>
+  </Accordion>
+
+{" "}
+<Accordion title="3. Build chart">
+  
+  For your first chart select a **Grouped bar**, and choose **country** as the x-axis and  **demand** as the y-axis.
+  the corresponding x- & y-axis
+  
+  For future charts see [Chart Configuration](/charts/create-chart) for all chart settings. 
+  
+  
+
+     <Frame>
+
+  <img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/chart-builder-dark.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/chart-builder-dark.png"
+  alt="Hero Dark"
+/>
+     </Frame>
+
+</Accordion>
+
+  <Accordion title="4. Add chart filters">
+  To add chart filters 
+    You can set up filters by clicking on **configure filters** and selecting the relevant column and filter type.
+
+
+    <Steps>
+      <Step title="Select Table ">
+      **1. Raw backing tables:** filter any column in the raw data that is powering this specific chart
```

Comment:
Lets use consistent wording. Seems like we use table everywhere, if so, replace "raw data" with table. Also strictly speaking, (1) isn't really raw backing tables. They are just selected backing tables

I would say replace it with:
**1. Selected backing tables:** The list of selected backing tables in Step 1 is surfaced here. Filter any column in any selected backing table(s) that is powering this specific chart